bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -98,10 +100,10 @@ spec:
               mountPath: /tmp/certs
             - name: pgpool-certificates
               mountPath: /opt/bitnami/pgpool/certs
-      {{- end }}
       {{- if .Values.pgpool.initContainers }}
       {{- include "common.tplvalues.render" (dict "value" .Values.pgpool.initContainers "context" $) | nindent 8 }}
       {{- end }}
+      {{- end }}
```

Comment:
Moving this `{{- end}}` would nest the `pgpool.initContainers` under the `pgpool.tls.enabled`, causing the first to not be rendered if the later is not enabled.

```
      {{- if .Values.pgpool.tls.enabled }}
        - name: init-chmod-data
          image: {{ template "postgresql-ha.volumePermissions.image" . }}
              mountPath: /tmp/certs
            - name: pgpool-certificates
              mountPath: /opt/bitnami/pgpool/certs
      {{- if .Values.pgpool.initContainers }}
      {{- include "common.tplvalues.render" (dict "value" .Values.pgpool.initContainers "context" $) | nindent 8 }}
      {{- end }}
      {{- end }}
```

Reply:
Thank you for the feedback. This should be resolved now. Please let me know if you find any additional issues!